src/get-api-credentials.js

```diff
@@ -0,0 +1,19 @@
+import { ProjectCredentialsConfig } from 'sphere-node-utils'
+import Promise from 'bluebird'
+
+const getApiCredentials = (projectKey, accessToken) => {
+  if (!projectKey)
+    return Promise.reject(new Error('Project Key is needed'))
+
+  if (accessToken)
+    return Promise.resolve({ project_key: projectKey })
+
+  return ProjectCredentialsConfig.create()
```

Comment:
The utils function instantiates a new object by itself when called `create` on it.